de.metas.adempiere.adempiere/base/src/main/java/de/metas/translation/api/IElementTranslationBL.java

```diff
@@ -48,9 +48,21 @@
 
 	/**
 	 * Update Fields translation for the fields that have the element given as parameter as AD_Name_ID, delete translation if AD_Name was deleted
-	 * 
+	 *
 	 * @param ad_Element_ID
 	 */
 	void updateFieldTranslationsFromAD_Name(int ad_Element_ID);
 
+	void updateWindowTranslationsFromElement(int elementId);
```

Comment:
always use object IDs when possible/available.
i.e. in this case, use AdElementId